libp2p/crypto/crypto.nim

```diff
@@ -69,7 +69,7 @@ when supported(PKScheme.Secp256k1):
 # We are still importing `ecnist` because, it is used for SECIO handshake,
 # but it will be impossible to create ECNIST keys or import ECNIST keys.
 
-import ecnist, bearssl
+import ecnist, bearssl/rand, bearssl/hash as bhash, bearssl/abi/intx
```

Comment:
`intx` should be fixed in bearssl instead - ie it should be imported by any `abi/bearss_xxx` that requires access to `int` symbols - what was the error message exactly?